core/iwasm/common/wasm_runtime_common.c

```diff
@@ -1743,6 +1743,29 @@ wasm_runtime_finalize_call_function(WASMExecEnv *exec_env,
 }
 #endif
 
+bool
+clear_wasi_proc_exit_exception(WASMModuleInstanceCommon *module_inst_comm)
+{
+#if WASM_ENABLE_LIBC_WASI != 0
+    WASMModuleInstance *module_inst = (WASMModuleInstance *)module_inst_comm;
+
+    bh_assert(module_inst_comm->module_type == Wasm_Module_Bytecode
+              || module_inst_comm->module_type == Wasm_Module_AoT);
+
+    const char *exception = wasm_get_exception(module_inst);
```

Comment:
Had better declare the variable at the beginning of the function, a concern is that old version compiler might report warning.

Reply:
Done, thanks